python/lsst/ts/atmcssimulator/schemas/logevent_allAxesInPosition.json

```diff
@@ -0,0 +1,20 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "type": "object",
+    "properties": {
+        "id": {
+            "enum": [
+                "allAxesInPosition"
+            ]
+        },
```

Comment:
The data type of "id" of event wound be "string" instead of "enum". Therefore, it would be `"id": "allAxesInPosition"` This applies to all the event related json.

Reply:
In this case I was trying to make sure that only that particular string can be chosen. I found a better way to do this via the `const` type so I opted for that.